lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -130,6 +130,13 @@ struct MemberVisitor {
       // connection for the port.
       if (!expr) {
         auto *port = con->port.as_if<PortSymbol>();
+        for (const auto &existPort : moduleLowering->ports)
+          if (port->name == existPort.ast.name) {
+            moduleLowering->portsBySyntaxNode.insert(
+                {con->port.getSyntax(), &existPort.ast});
+            port = &existPort.ast;
+          }
```

Comment:
Sorry, modulerLowering is created in convertModuleHeader(), and this interface only handle isntanse body( = InstNode.body). However the ports information is in instNode.getPortConnections(). I am not sure if this information belongs to module header. 
Easily I can show you what it will be.